.github/actions/security-controls/action.yml

```diff
@@ -0,0 +1,23 @@
+name: "Generate github issues"
```

Comment:
Could you make this more specific - something like `Generate ITSG-33 control GitHub issues` maybe?

Reply:
Done!